processor/intervalprocessor/testdata/summaries_are_aggregated/input.yaml

```diff
@@ -0,0 +1,63 @@
+resourceMetrics:
+  - schemaUrl: https://test-res-schema.com/schema
```

Comment:
I have a feeling that this is copied from some existing files, _but_ I wanted to mention that `example.com` is the domain to be used in tests as per RFC 2606.

Reply:
I'm merging as is, but if you feel like fixing this, open a follow-up PR.